packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -140,14 +140,18 @@ const Identity = (x) => x;
 // Safari, on iOS, supports touch action since v13.
 // Over 80% of the iOS phones are compatible
 // in August 2020.
+// Utilizing the CSS.supports method to check if touch-action is supported.
+// Since CSS.supports is supported on all but Edge@12 and IE and touch-action
+// is supported on both Edge@12 and IE if CSS.supports is not available that means that
+// touch-action will be supported
 let cachedSupportsTouchActionNone;
 function doesSupportTouchActionNone() {
   if (cachedSupportsTouchActionNone === undefined) {
-    const element = document.createElement('div');
-    element.style.touchAction = 'none';
-    document.body.appendChild(element);
-    cachedSupportsTouchActionNone = window.getComputedStyle(element).touchAction === 'none';
-    element.parentElement.removeChild(element);
+    if (typeof CSS !== 'undefined' && typeof CSS.supports === 'function') {
+      cachedSupportsTouchActionNone = CSS.supports('touch-action', 'none');
+    }
+
+    cachedSupportsTouchActionNone = true;
```

Comment:
We need to set this in an `else` branch. Otherwise it always overrides the actual value.

Good news is that we can actually test this behavior in browserstack.